README.md

```diff
@@ -1,17 +1,21 @@
 # PYMMH3 - a pure python MurmurHash3 implementation.
 
-[![Build Status](https://travis-ci.org/wc-duck/pymmh3.svg?branch=master)](https://travis-ci.org/wc-duck/pymmh3)
+[![Build Status](https://travis-ci.org/swapniel99/pymmh3.svg?branch=master)](https://travis-ci.org/swapniel99/pymmh3)
 
 ## About:
 This is a pure python implementation of the murmur3 hash algorithm <https://code.google.com/p/smhasher/wiki/MurmurHash3>
 written for the times when you do not want to muck about building and installing c-modules etc.
 
-As this is pure python performance is FAR from optimal and if that is required a real c-module such as the
-one found here at <https://pypi.python.org/pypi/mmh3/2.0> is absolutely suggested!
+As this is pure python performance is FAR from optimal in CPython and if that is required a real c-module such as the
+one found here at <https://pypi.python.org/pypi/mmh3/2.3.1> is absolutely suggested!
+
+However this module performs extremely well in case of pypy(<http://pypy.org>), an alternative JIT version of python.
```

Comment:
C module(https://github.com/hajimes/mmh3) performs extremely well in cpython.
But in pypy it is the other way round.